### DISCUSSION TITLE
SAML2 not working

### DISCUSSION THREAD

**Discussion by pat0s** [requarks/wiki#6667] [Help / Questions] [answered]
## Description
I set up authentication using the SAML 2.0 module. I checked SAML Request and SAML Response everything looks OK. I am redirected to our organization's login page where I can login successfully. But after redirection back to wiki the following error occurs. I don't think this error is related to the [passport-saml](https://github.com/node-saml/passport-saml) package. 
![error](https://github.com/requarks/wiki/assets/58843354/482423f9-318c-4a00-b110-7543dfb50e01)

There are no errors in the logs after the wiki startup or after authentication.
`Aug 23 08:02:57 js-doc-23 node[426348]: 2023-08-23T06:02:57.009Z [MASTER] #033[32minfo#033[39m: Authentication Strategy SAML 2.0: [ OK ]`

**Comment by NGPixel** [comment, marked as answer]
Make sure you're using the callback URL provided at the bottom of the SAML config page.

**Reply by pat0s** [reply]
Thanks, the callback URI was different.

**Reply by pat0s** [reply]
@NGPixel Why has Wiki JS changed the callback URI? I set up SAML authentication with my colleague. I double-checked it and it was different. 

Fortunately, everything is working now.

**Reply by NGPixel** [reply]
The URL doesn't change... You most likely removed and added the strategy back, which generated a new ID...

**Reply by pat0s** [reply]
Yes, I also think the URI doesn't change. I just want to make sure.

PEBKAC 🥲
